Dialogue/apc_chat.csv

```diff
@@ -7304,22 +7304,22 @@ IncidentJsApcBegin#99,"""/ci1 1 おや、良いところに！<br>すまぬが
 IncidentJsApcFailure#99,"""全ては拙者の力不足が原因……<br>先に退かせてもらうでござる"""
 IncidentJsApcSuccess#99,"""/ci2 1 これにて一件落着、でござるな。<br>助太刀感謝するでござるよ"""
 IncidentJsApcBeginEvo#97,"""/ci1 2 これにて終了……とは<br>いかないようでござるな"""
-EnemyWeak#110,"""ケッ、雑魚ばっかかよ！"""
-EnemyLarge#110,"""ハッ、図体だけはでかいようだな！"""
-EnemyBoss#110,"""少しは手応えのありそうなヤツが<br>出てきたじゃねえか！"""
-EnemyRare#110,"""んん？<br>ありゃあ、珍しいヤツか？"""
-EnemyBoost#110,"""ちっ、メンドクセェ感じの<br>ヤツが出てきてやがるな……！"""
-DestroyEnemy1#110,"""雑魚が！"""
-DestroyEnemy2#110,"""邪魔だ！"""
-DestroyEnemy3#109,"""うざってぇんだよ！"""
-CondFreeze#110,"""凍結だと……っざけんな！"""
-CondBurn#110,"""燃焼だと……っざけんな！"""
+EnemyWeak#110,"""Kuh, weak like a small fish"""
+EnemyLarge#110,"""Happy?! nothing more than the body"""
+EnemyBoss#110,"""I guess these guys finally came out"""
+EnemyRare#110,"""Whut? Awh, it's a rare person?"""
```

Comment:
Whut? or What?
And the spell checker does not like Awh, so you may have add to _aspell/PSO2.en_Chat.dict